bpf/lib/dbg.h

```diff
@@ -220,7 +220,7 @@ static __always_inline void cilium_dbg_capture2(struct __ctx_buff *ctx, __u8 typ
 	__u64 cap_len = min_t(__u64, TRACE_PAYLOAD_LEN, ctx_len);
 	struct debug_capture_msg msg = {
 		__notify_common_hdr(CILIUM_NOTIFY_DBG_CAPTURE, type),
-		__notify_pktcap_hdr(ctx_len, (__u16)cap_len),
+		__notify_pktcap_hdr(ctx_len, (__u16)cap_len, NOTIFY_CAPTURE_VER),
```

Comment:
Should we add a use a different constant here than re-using the one for notify messages?
```suggestion
		__notify_pktcap_hdr(ctx_len, (__u16)cap_len, DBG_VER),
```

Reply:
My reading is this is already generic for any notify message (trace, policy, debug, and drop).

Expanded on the comment to clarify this point.

Reply:
Sure, but if the same constant is used for all the message types I don't see how they can evolve independently as the commit message suggest. Not blocking though, extracting it can pave the way for future changes.

Reply:
future commits would create a new constant: e.g. `bpf/lib/drop.h`

```
#define NOTIFY_DROP_VER = 2;
...

	msg = (typeof(msg)) {
		__notify_common_hdr(CILIUM_NOTIFY_DROP, (__u8)error),
		__notify_pktcap_hdr(ctx_len, (__u16)cap_len, NOTIFY_DROP_VER),
		...
```